symfony/client/src/core/util/helper/datefns.ts

```diff
@@ -331,6 +333,14 @@ const guessTimezone = () => {
   };
 };
 
+const getStandardTimezone = (timezoneOffset: number) => {
```

Comment:
can't see unit tests for this and above method

Reply:
added